<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>250</width>
     <height>190</height>
    </rect>
   </property>
   <property name="title">
    <string>1번</string>
   </property>
   <widget class="QLineEdit" name="statusEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>210</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="backlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>250</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>backlabel</zorder>
   <zorder>statusEdit</zorder>
   <zorder>timeLabel</zorder>
  </widget>
  <widget class="QTableWidget" name="WaitingList">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>60</y>
     <width>320</width>
     <height>610</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>대기번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>대기인원</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>상태</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>250</width>
     <height>190</height>
    </rect>
   </property>
   <property name="title">
    <string>2번</string>
   </property>
   <widget class="QLineEdit" name="statusEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>210</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="backlabel_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>250</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>250</width>
     <height>190</height>
    </rect>
   </property>
   <property name="title">
    <string>3번</string>
   </property>
   <widget class="QLineEdit" name="statusEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>210</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="backlabel_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>250</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnUser">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>user</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
